operator/internal/manifests/openshift/internal/dashboards/build.go

```diff
@@ -0,0 +1,61 @@
+package dashboards
+
+import (
+	"embed"
+	"fmt"
+	"io/fs"
+	"path"
+)
+
+const (
+	staticDir                       = "static"
+	lokiStackChunkDashboardFile     = "grafana-dashboard-lokistack-chunks.json"
+	lokiStackReadsDashboardFile     = "grafana-dashboard-lokistack-reads.json"
+	lokiStackWritesDashboardFile    = "grafana-dashboard-lokistack-writes.json"
+	lokiStackRetentionDashboardFile = "grafana-dashboard-lokistack-retention.json"
+	lokiStackDashboardRulesFile     = "grafana-dashboard-lokistack-rules.json"
+)
+
+//go:embed static/*.json
+var lokiStackDashboards embed.FS
+
+// ReadDashboards returns the byte slices one for each LokiStack dashboard
+// from the embedded filesystem or an error.
+func ReadDashboards() (map[string][]byte, error) {
```

Comment:
Although all errors thrown here are from from the `fs` package reading from `embed` and in turn rather close to impossible, I would like to reconcile if they happens instead for example to panic in `init`. WDYT?

Reply:
If I am not forgetting anything any failures that would lead to a `panic` in this case could be handled by having unit tests. We should be able to do this safely during initialization.